### PR TITLE
Update milter protocol to 6

### PR DESCRIPTION
**Pull request by 9600**
Seeing DKIM header signature failures which are fixed by setting Postfix parameter milter_protocol value to 6. Possibly related to: https://marc.info/?l=postfix-users&m=157746682807997&q=mbox .

### CODE REVIEW

**Comment by hemberger** [comment]
The default `milter_protocol` value is 6 as of postfix 2.6 (released in 2009), so this seems like a sensible change to me. (see https://www.postfix.org/MILTER_README.html#version)

**Comment by 9600** [comment]
Sounds good. Just confirmed that the issue is also resolved by not setting a version and going with the default.

**Comment by wader** [comment]
@hemberger use default ok with you?

File: run [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # DKIM config
 dkimConfig()
 {
-    postconf -e milter_protocol=2
```
[the remark this answers is not in the document]
Comment (wader) [reply run:6]: Oh maybe i commented on the wrong line, was about KeyTable

**Comment by hemberger** [comment]
> @hemberger use default ok with you?

Sounds good to me!

**Comment by wader** [comment]
Thanks again, my own use of this image is quite basic so appreciate all help

**Comment by 9600** [comment]
You're welcome and thanks for the handy image!

**Pull request merged**